compiler/resolution/wrappers.cpp

```diff
@@ -2659,13 +2666,17 @@ static void buildFastFollowerCheck(bool                  isStatic,
   returnTmp->addFlag(FLAG_EXPR_TEMP);
   returnTmp->addFlag(FLAG_MAYBE_PARAM);
 
-  if (isStatic == true) {
-    fnName          = "chpl__staticFastFollowCheck";
+  if (checkType == CAN_HAVE_FF) {
+    fnName          = "chpl__canHaveFastFollowers";
 
     checkFn         = new FnSymbol(fnName);
     checkFn->retTag = RET_PARAM;
+  } else if (checkType == STATIC_FF_CHECK) {
+    fnName          = "chpl__staticFastFollowCheck";
 
-  } else {
+    checkFn         = new FnSymbol(fnName);
+    checkFn->retTag = RET_PARAM;
+  } else if (checkType == DYNAMIC_FF_CHECK) {
```

Comment:
Nit, but might be nice to have something here in case other checks are added in the future to make sure we don't miss adding to the if/else branch. Maybe just have this be an else branch and assert DYNAMIC_FF_CHECK or convert to a switch statement?